torch/onnx/symbolic_opset10.py

```diff
@@ -178,3 +179,67 @@ def flip(g, input, dims):
 
 def fmod(g, input, other):
     return g.op("Mod", input, other, fmod_i=1)
+
+
+@parse_args('v', 'v', 'v', 'i', 'i', 'i', 'v', 'i')
+def embedding_bag(g,
+                  embedding_matrix,
+                  indices,
+                  offsets,
+                  scale_grad_by_freq,
+                  mode,
+                  sparse,
+                  per_sample_weights,
+                  include_last_offset):
+    from torch.onnx.symbolic_opset9 import size, div
+
+    if not sym_help._is_none(per_sample_weights):
+        return sym_help._onnx_unsupported('embedding_bag  with per_sample_weights')
+
+    if offsets.type().sizes() is not None:
+        if indices.node().kind() != 'prim::Param':
```

Comment:
I assume this is the 2d input case. But why can we tell from this condition? `if indices.node().kind() != 'prim::Param':`

Also worth noting here why while the doc mentions `offsets` should be None, but still have value here.

> If input is 2D of shape (B, N),
> 
> it will be treated as B bags (sequences) each of fixed length N, and this will return B values aggregated in a way depending on the mode. **offsets is ignored and required to be None in this case.**
>

Reply:
I actually updated this to check for onnx::Reshape type. All inputs (1D and 2D) are reshaped to 1D in functional.py. And for 2D inputs offsets is set to:
torch.arange(0, indices.numel(), indices.size(1))